sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -294,3 +292,22 @@ export interface Assessment {
   totalProvincialContribution: number;
   otherAllowableCost: number;
 }
+
+/**
+ * Interface for PartTime assessment payload.
+ */
+export interface PartTimeAssessment extends BaseAssessment {
+  schoolFees: number;
+  miscellaneousCost: number;
+  totalFamilyIncome: number;
```

Comment:
totalFamilyIncome should be the same as FT

Reply:
In FT we have 

  studentTotalFederalContribution,
  studentTotalProvincialContribution,
  partnerAssessedContribution,
  parentAssessedContribution,
  
  For PT, we have it more direct, So i used totalFamilyIncome to show the contribution in the NOA page

Reply:
ah, lets connect